doc/source/reference/api/api.rst

```diff
@@ -721,7 +721,6 @@ Function List
 
 .. autosummary::
 
-   ~yt.convenience.load
```

Comment:
wasn't this removed in yt-project/yt#3066

Reply:
apparently yeah

Reply:
since the change is identical, it's not causing a conflict, but indeed this is now redundant.